tests/test_svg.py

```diff
@@ -257,6 +257,31 @@ def color_function(data, on):  # noqa ARG001
                     )
                 )
 
+        # Test Export of gus vertices
+        vertices = c.gus.Vertices(spline.cps)
+        vertices.vertex_data["field"] = c.np.linspace(
+            0, 1, spline.cps.shape[0]
+        )
+
+        with tempfile.TemporaryDirectory() as tmpd:
+            tmpf = c.to_tmpf(tmpd)
+            c.splinepy.io.svg.export(
+                tmpf,
+                vertices,
+                indent=False,
+                box_margins=0.4,
+                background_c="black",
```

Comment:
```suggestion
                background="black",
```
`_c` is `_c`onfusing

Reply:
Is similar to "knot_c", "control_point_c"... if you really want I will change this

Reply:
I would like the change, unless there're other sub-options regarding background that I am missing. Good point though.